ui/ui-components/pages/model/[modelId]/group/[groupId]/rules.tsx

```diff
@@ -485,18 +486,17 @@ export default function Page(props) {
             Add Rule
           </Button>
           &nbsp;
-          {process.env.GROUPAROO_UI_EDITION !== "config" && (
-            <LoadingButton
-              disabled={loading}
-              variant="outline-dark"
-              size="sm"
-              onClick={async () => {
-                await getCounts(false);
-              }}
-            >
-              Count Potential Group Members
-            </LoadingButton>
-          )}
+          <LoadingButton
+            disabled={loading}
+            variant="outline-dark"
+            size="sm"
+            hideOn={["config"]}
+            onClick={async () => {
+              await getCounts(false);
+            }}
```

Comment:
```suggestion
            onClick={() => getCounts(false)}
```
This shouldn't need to care about async/await since there's nothing happening after.